src/testdrive/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "mz-testdrive"
 description = "Integration test driver for Materialize."
-version = "0.0.0"
+version = "0.70.0-dev"
```

Comment:
If this is going to be versioned going forward, please add this to `bin/bump-version` , line 32

Reply:
Oh you did that already, thanks!